Cython/Compiler/Parsing.py

```diff
@@ -2920,6 +2920,8 @@ def p_c_simple_declarator(s, ctx, empty, is_type, cmethod_flag,
                             fatal=False)
                 name = name + ' ' + op
                 s.next()
+        name = EncodedString(name)
+        if cname is not None: cname = EncodedString(cname)
```

Comment:
Could this be moved into `p_opt_cname`?

Reply:
It looks to be already be in cname. This is in the part I've now taken out of the PR so I'll investigate when preparing the other one.